### DISCUSSION TITLE
Google Play version

### DISCUSSION THREAD

**Discussion by JimmyS83** [octoshrimpy/quik#193] [General]
Hello,

Any plans for pushing this app to Google Play?

For some reason, valid QKSMS version there is 3.9.2.  Outdated, but I never had an issue with it. Reading some information regarding newer versions (however, allegedly through f-droid, not from Gplay or Github) - https://forum.f-droid.org/t/qksms-compromised/16741/30, I am not sure, if thats anyhow related to not having latest QKSMS @Gplay.

**Comment by octoshrimpy** [comment]
Too Many variables and not enough information to rule qksms as the culprit. 

See: https://youtu.be/wVyu7NB7W6Y

**Reply by JimmyS83** [reply]
> Too Many variables and not enough information to rule qksms as the culprit

I agree. But still, I would love to see nowadays version in Google Play - meaning Quik in this case :)
